recipes/cpplint/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "cpplint" %}
+{% set version = "1.4.4" %}
+{% set sha256 = "05f879aab5a04307e916e32afb547567d8a44149ddc2f91bf846ce2650ce6d7d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/cpplint/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Please add ``noarch: python``

Reply:
Good, I'll try. That gave me errors previously with cpplint.exe being a binary (which would surprise me). Let's see.

Reply:
if you have a binary which needs to be executed in a specific platform, it cannot be ``noarch``
I thought that was a pure python project

Reply:
Thanks. it works now.